testing/asserts.ts

```diff
@@ -766,8 +771,11 @@ export async function assertRejects<E extends Error = Error>(
     }
     doesThrow = true;
   }
-  if (!doesThrow) {
-    msg = `Expected function to throw${msg ? `: ${msg}` : "."}`;
+  msg = msg ? `: ${msg}` : ".";
```

Comment:
I don't think we should be modifying the msg argument.

Reply:
It was already being modified, I only did that not to duplicate `msg ? `: ${msg}` : "."` and both error messages. But I agree that there is a better way to do that and keep the function as pure as possible 😄

Reply:
Good point on it already being modified. Didn't notice that. I guess the modification part is fine. You can just move that line up to the top I guess, that way you can make use of it in both the catch and the last assertion at the end of the function.

Reply:
I removed the modification on my last push, would it be alright? If so I'll do it on `assertThrows` too, if not I'll revert this part.